src/components/NavbarFooter/Navbar/index.jsx

```diff
@@ -21,10 +21,10 @@ import {
 } from "@chakra-ui/react";
 import {BellIcon, ChevronDownIcon, HamburgerIcon, Icon} from "@chakra-ui/icons";
 import {FaSun, FaMoon } from 'react-icons/fa';
-import NotificationList from "../NotificationList";
-import LoginPopup from "../LoginPopup";
-import {useUser} from "../../contexts/useUser";
-import {useAuth} from "../../contexts/useAuth";
+import NotificationList from "../../NotificationList";
+import LoginSignupPopup from "../../LoginSignup/LoginSignupPopup";
+import {useUser} from "../../../contexts/useUser";
+import {useAuth} from "../../../contexts/useAuth";
```

Comment:
I think this folder should not be called Navbar Footer since it contains the navbar, footer and page template

Reply:
what do you think it should be called then? PageTemplate?

Reply:
@HGoyal09 renaming folder to PageLayout as per our discussion